packages/pwa-kit-create-app/package.json

```diff
@@ -35,10 +35,10 @@
     "semver": "^7.5.2",
     "shelljs": "^0.8.5",
     "slugify": "^1.6.5",
-    "tar": "^6.2.1"
+    "tar": "^6.2.1",
+    "@salesforce/pwa-kit-dev": "4.0.0-dev"
```

Comment:
We talked about this IRL. I think since I (Ben) am working on the stand alone extensibility package that will be a better place to put that regex, that for now we simply react a duplicate regex and update it later.